packages/ui/src/views/pull-request/pull-request-list-description.tsx

```diff
@@ -0,0 +1,50 @@
+import { Icon } from '../../components/icon'
+import { Text } from '../../components/text'
+
+export const PullRequestListDescription = ({
+  reviewRequired,
+  number,
+  // tasks,
+  author,
+  source_branch,
+  timestamp
+}: {
+  number: number
+  reviewRequired?: boolean
+  tasks?: number
+  author: string
+  source_branch: string
```

Comment:
Can we keep to camelcase for prop names for consistency?
```suggestion
  sourceBranch: string
```

Reply:
done